ci-operator/config/redhat-developer/odo/redhat-developer-odo-master.yaml

```diff
@@ -5,7 +5,6 @@ resources:
     limits:
       memory: 6Gi
     requests:
-      cpu: 100m
```

Comment:
@mohammedzee1000 Removed as discussed in our bjn call with @kadel

Reply:
did you tru to update pr from a branch that was already used without rebasing

Reply:
no, Anyway now its fine.